diff -Nur planet-venus/diveintomark_20080214/index.html.tmpl planet-venus.new/diveintomark_20080214/index.html.tmpl
--- planet-venus/diveintomark_20080214/index.html.tmpl	2008-02-14 02:59:27.000000000 +0000
+++ planet-venus.new/diveintomark_20080214/index.html.tmpl	2008-02-15 02:04:17.000000000 +0000
@@ -6,7 +6,7 @@
 <title><TMPL_VAR name></title>
 <link rel=alternate type=application/atom+xml href=/atom.xml title=Feed>
 <style type=text/css media=screen>
-html{background:#FFF url(/i/_.png) repeat-x top left}
+html{background:#FFF}
 html,a{color:#333333}
 body{font:normal medium 'Gill Sans',Optima,Verdana,sans-serif;margin:40px auto 1em auto;width:728px;line-height:180%}
 #footer{clear:both;text-align:center;font-size:small}
@@ -46,4 +46,4 @@
 </ul>
 </div>
 </body>
-</html>
\ No newline at end of file
+</html>
